qa/full-cluster-restart/build.gradle

```diff
@@ -58,18 +59,4 @@ for (Version bwcVersion : BuildParams.bwcVersions.indexCompatible) {
   tasks.register(bwcTaskName(bwcVersion)) {
     dependsOn tasks.named("${baseName}#upgradedClusterTest")
   }
-}
```

Comment:
Because these are purely test projects I wonder if we could just make the changes we want here in the end? I get being stuck with test artifacts for things like `server` because no one has time to untangle it. But for this one and the next one maybe its pretty simple?

I guess it doesn't make sense to block the change on this though. But maybe it'd be worth doing for the easy ones

Reply:
We can look into this in the future, but I would not want to mix this into this PR